tests/posix/common/src/clock.c

```diff
@@ -105,6 +111,9 @@ ZTEST(clock, test_gettimeofday)
 	/* Validate gettimeofday API */
 	zassert_ok(gettimeofday(&tv, NULL));
 	zassert_ok(clock_gettime(CLOCK_REALTIME, &rts));
+#ifdef CONFIG_POSIX_THREAD_CPUTIME
+	zassert_ok(clock_gettime(CLOCK_THREAD_CPUTIME_ID, &ts));
+#endif /* CONFIG_POSIX_THREAD_CPUTIME */
```

Comment:
Why are we testing `clock_gettime()` inside `test_gettimeofday`, which is supposed to test `gettimeofday()`?

Reply:
Good catch as well. This bit can be removed